src/main/java/cubicchunks/server/ServerCubeCache.java

```diff
@@ -76,14 +76,15 @@
 	private Queue<CubeCoords> cubesToUnload;
 	private Queue<ChunkPos> columnsToUnload;
 
+	// TODO: Use a better hash map!
 	private Map<CubeCoords, Cube> cubemap = new HashMap<>();
 
 	private ICubeGenerator   cubeGen;
 
 	public ServerCubeCache(ICubicWorldServer worldServer, ICubeGenerator cubeGen) {
 		super((WorldServer) worldServer,
 				worldServer.getSaveHandler().getChunkLoader(worldServer.getProvider()), // forge uses this in
-				null); // safe to null out IChunkGenerator
+				null); // safe to null out IChunkGenerator (Note: lets hope mods don't touch it, ik its public)
```

Comment:
Why don't we just provide something here? Before your pull request we passed something, so apparently we have the relevant implementation.